packages/xo-web/src/common/intl/messages.js

```diff
@@ -1008,6 +1008,9 @@ const messages = {
 
   // ----- VM general tab -----
   noToolsDetected: 'No Xen tools detected',
+  managementAgentDetected: 'Management agent {version} detected',
```

Comment:
`Management agent version {version} detected`

Reply:
`version` cannot exists.  See https://github.com/vatesfr/xen-orchestra/pull/5007/files#diff-0e365e7b72e3917a357e2e7c79f176cfR69

Reply:
When `pvDriversVersion` can be 0?

Reply:
When, i don't know, it depends on installed `XCP-ng tools` . The hiding of the version 0 was discussed with samuel which told me that it can confuse users.